### PR TITLE
Extract naming module

### PR DESCRIPTION
**Pull request by LeonMatthesKDAB**
This naming module could later be extracted into another crate that may be shared with other crates that generate code for CXX.

### CODE REVIEW

File: crates/cxx-qt-gen/src/generator/cpp/mod.rs [hunk 1]
```diff
@@ -73,6 +73,55 @@ impl GeneratedCppBlocks {
     }
 }
 
+mod utils {
```
Comment (ahayzen-kdab) [inline crates/cxx-qt-gen/src/generator/cpp/mod.rs:76]: any reason for this not being in a `cpp/utils.rs` ?

Comment (LeonMatthesKDAB) [reply crates/cxx-qt-gen/src/generator/cpp/mod.rs:76]: Just thought it's a bit too small to deserve it's own file, but don't mind either way :)
Will move into a file :+1:

Comment (LeonMatthesKDAB) [reply crates/cxx-qt-gen/src/generator/cpp/mod.rs]: Done

**Review by ahayzen-kdab — APPROVED** [review]
LGTM

**Pull request merged**